### PR TITLE
Add background color to grid to prevent animation overflow

### LINKED ISSUE #10664: Funky scroll bars in 1.10's settings UI
### Windows Terminal version (or Windows build number)

Microsoft Windows 10 Pro for Workstations 10.0.19042.1110 (2009, 20H2) WindowsTerminal: 1.10.1933.0

### Other Software

_No response_

### Steps to reproduce

Start Windows Terminal 1.10.1933.0 ... open the settings UI ... go to "Defaults" or any profile ... observe the scroll bar

### Expected Behavior

Not what's pictured below

### Actual Behavior

image

### PR DESCRIPTION
**Pull request by mimvdb**
## Summary of the Pull Request
Add an explicit background color to part of the settings UI to prevent animation overflow. The previous solution (adding a ScrollViewer) caused problems.

## References
#10619 adds a ScrollViewer for one of the issues in #10609

## PR Checklist
* [x] Closes #10664
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Documentation updated. If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
* [ ] Schema updated.
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Validation Steps Performed
Visually confirmed the animation doesn't overflow, changed the theme and confirmed the colors are responsive. Confirmed the extra scrollbar is gone.

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/MainPage.xaml [hunk 1]
```diff
@@ -116,13 +116,14 @@
...
-            </ScrollViewer>
+            <Frame x:Name="contentFrame"
+                   Grid.Row="0"/>
+            <!-- Explicitly set the background color on grid to prevent the navigation animation from overflowing it -->
             <Grid Grid.Row="1"
                   Height="100"
                   BorderBrush="{ThemeResource SystemBaseLowColor}"
-                  BorderThickness="0,1,0,0">
+                  BorderThickness="0,1,0,0"
+                  Background="{ThemeResource SystemAltHighColor}">
```
Comment (mimvdb) [inline src/cascadia/TerminalSettingsEditor/MainPage.xaml:126]: This color looks right to me in dark and light themes, but I'm not familiar with which colors are actually supposed to be used. My first attempt `SystemColorWindowColor` turned out not to support themes.

Comment (leonMSFT) [reply src/cascadia/TerminalSettingsEditor/MainPage.xaml]: Would it also fix the issue if you make the background `Transparent`?

Comment (mimvdb) [reply src/cascadia/TerminalSettingsEditor/MainPage.xaml]: It does not

Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsEditor/MainPage.xaml]: Yea I'll be real, I have no idea why some actually work when theme-switched and others don't. I double checked light & dark & HC and this seems to work with them all.

**Review by zadjii-msft — APPROVED** [review]

**Review by leonMSFT — APPROVED** [review]

**Pull request merged**